### ISSUE TITLE
I don't know how to use the command ocropus-rtrain with the -t argument to do the error estimation.

### ISSUE THREAD

**Issue by Kevinnest** [tmbdev/ocropy#159]
I can produce the ocropus models successfully, based on the training data. But I don't know how to use the -t argument correctly to do the error estimation.

**Comment by zuphilip** [comment]
It looks for me that the `-t`/`--test` option does not have any influence in the following code of `ocropus-rtrain`: [it will be saved in a variable `tests` but not used afterwards anywhere](https://github.com/tmbdev/ocropy/blob/8c86aa85baaf7afa8ee83948fc40101fac77003d/ocropus-rtrain#L111-L114). Moreover, also in the history I couldn't find any occurrence of using this option. Can someone else confirm that this option is missing and was never there?

It is possible to observe the error rate during training on the training set itself, e.g. just look at the example in `run-rtrain` (the bottom diagram shows then the error rate as red line).

**Comment by zuphilip** [comment]
For testing against a separated training set, you can save the model more frequently with ` -F SAVEFREQ, --savefreq SAVEFREQ` option and then use `ocropus-rpred`. I guess it is important to do the same line normalization steps for training and prediction.

**Comment by Kevinnest** [comment]
Thanks for replying.  I was just confused about the usage of the -t option, because I noticed the fact, as you said, the variable _tests_ is not used afterwards. I was hoping to find an easy way to test the accuracy of each model. Now I decided to calculate the accuracy by myself.

**Comment by kba** [comment]
Yeah, that's a missing feature. It is implemented in clstm (https://github.com/tmbdev/clstm/blob/master/clstmocrtrain.cc#L193-L207), @jbaiter's pyclstm bindings script (https://github.com/jbaiter/clstm/blob/cython/pyclstm_cli/train.py#L127-L133) and kraken (https://github.com/mittagessen/kraken/blob/master/kraken/ketos.py#L149-L153).

**Comment by jbins** [comment]
Had this same issue.

**Comment by gjerome5** [comment]
How can we create our model with training data?
